qos/cometbft/store.go

```diff
@@ -39,12 +39,12 @@ func (es *EndpointStore) Select(availableEndpoints []protocol.EndpointAddr) (pro
 
 	filteredEndpointsAddr, err := es.filterValidEndpoints(availableEndpoints)
 	if err != nil {
-		logger.Warn().Err(err).Msg("error filtering endpoints")
+		logger.Error().Err(err).Msg("error filtering endpoints")
 		return protocol.EndpointAddr(""), err
 	}
 
 	if len(filteredEndpointsAddr) == 0 {
-		logger.Warn().Msg("all endpoints failed validation; selecting a random endpoint.")
+		logger.Error().Msg("SELECTING A RANDOM ENDPOINT because all endpoints failed validation.")
```

Comment:
This should not be an error-level log entry: even if we wait one full hydrator iteration before handling any service requests, endpoints in a new session may not have been previously observed.